src/include/network/routines.rb

```diff
@@ -947,6 +947,18 @@ def SetAllLinksUp
       ret
     end
 
+    def SetAllLinksDown
+      interfaces = GetAllInterfaces()
+      ret = !interfaces.empty?
+
+      interfaces.each do |ifc|
+        Builtins.y2milestone("Setting link down for interface %1", ifc)
+        ret = SetLinkDown(ifc) && ret
```

Comment:
so again logic says, that you try to down it all and if any failed, then continue with others.